README.md

```diff
@@ -53,18 +53,39 @@ Now that we've covered the basics of using `dotnew new`, lets move on to info fo
 If you're authoring templates, or interested in contributing to this repo, then you're likely interested in how to use the latest version of this experience.
 The steps required are outlined below.
 
-Step 1: Get the SDK for your platform from [dotnet/cli](https://github.com/dotnet/cli)
+## Prerequisites
+In order to work with this project, you will need the following.
+- [Visual Studio 2017](https://www.visualstudio.com/)
```

Comment:
Can you comment more on you statement of "making the full build Windows only at the moment"?